staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -244,6 +244,13 @@ message Mount {
     // - when set to true, readonly must be explicitly set to true, and propagation must be PRIVATE (0).
     // - (readonly == false && recursive_read_only == false) does not make the mount read-only.
     bool recursive_read_only = 8;
+    // Mount an image reference (image ID or digest), which is a special use
+    // case for OCI volume mounts. If this field is set, then host_path should be unset.
+    // The kubelet does an PullImage RPC and evaluates the returned PullImageResponse.image_ref
+    // value, which is then set to this field. Runtimes are expected to mount
+    // the image as required.
+    // Introduced in the OCI Volume Source KEP: https://kep.k8s.io/4639
```

Comment:
Does this require `readonly = true`?

Reply:
No, we only allow read only mounts per definition of the feature. But the kubelet can set the fields just to be sure.